plugins/package-managers/gradle-inspector/README.md

```diff
@@ -0,0 +1,45 @@
+# GradleInspector
+
+The [GradleInspector] is an alternative analyzer for projects that use the Gradle package manager. It is supposed to
+address [several][] [shortcomings][] of the "legacy" [Gradle] analyzer, but in order to not interfere with it, the
+[GradleInspector] is disabled by default.
+
+## Usage
+
+As the [GradleInspector] is disabled by default, it needs to be enabled explicitly. It is recommended to disable the
+"legacy" [Gradle] analyzer at the same time:
+
+    ort analyze -P ort.analyzer.enabledPackageManagers=GradleInspector -P ort.analyzer.disabledPackageManagers=Gradle
```

Comment:
As there is currently no way to add a package manager to the list of enabled package manager, mabye give a hint that the full list needs to be provided, for example with `=GradleInspector[,...]`.

Reply:
Ah, true, I've reworded the text.